bootstrap/sql/com.mysql.cj.jdbc.Driver/v004__create_db_connection_info.sql

```diff
@@ -116,3 +116,21 @@ DELETE FROM policy_entity;
 DELETE FROM field_relationship WHERE fromType IN ('role', 'policy') OR toType IN ('role', 'policy');
 DELETE FROM entity_relationship WHERE fromEntity IN ('role', 'policy') OR toEntity IN ('role', 'policy');
 ALTER TABLE role_entity DROP COLUMN defaultRole;
+
+UPDATE dbservice_entity
+SET json = JSON_INSERT(
+        JSON_REMOVE(json, '$.connection.config.metastoreHostPort'),
+        '$.connection.config.metastoreConnection',
+        JSON_OBJECT('metastoreHostPort', JSON_EXTRACT(json, '$.connection.config.metastoreHostPort'))
+    )
+where serviceType = 'DeltaLake'
+  and JSON_EXTRACT(json, '$.connection.config.metastoreHostPort') is not null;
+
+UPDATE dbservice_entity
+SET json = JSON_INSERT(
+        JSON_REMOVE(json, '$.connection.config.metastoreFilePath'),
+        '$.connection.config.metastoreConnection',
+        JSON_OBJECT('metastoreFilePath', JSON_EXTRACT(json, '$.connection.config.metastoreFilePath'))
+    )
+where serviceType = 'DeltaLake'
+  and JSON_EXTRACT(json, '$.connection.config.metastoreFilePath') is not null;
```

Comment:
I think we'll need to move this to v005

Reply:
@TeddyCr lets merge it and add a follow pr

Reply:
@harshach @TeddyCr I have already address the comments